src/features/class-static-initializer-blocks.md

```diff
@@ -0,0 +1,109 @@
+---
+title: 'Class static initializer blocks'
+author: 'Shu-yu Guo ([@_shu](https://twitter.com/_shu))'
+avatars:
+  - 'shu-yu-guo'
+date: 2021-03-19
+tags:
+  - ECMAScript
+description: 'JavaScript classes get dedicated syntax for static initialization.'
+---
+The new class static initializer block syntax lets developers gather code that should run once for a given class definition and put them in a single place. Consider the following example where a pseudo random number generator uses a static block to initialize an entropy pool once, when the `class MyPRNG` definition is evaluated.
+
+```js
+class MyPRNG {
+  seed;
+  constructor(seed) {
+    if (seed == undefined) {
+      if (MyPRNG.entropy_pool.length === 0) {
+        throw new Error("Entropy pool exhausted");
+      }
+      seed = MyPRNG.entropy_pool.pop();
+    }
+    this.seed = seed;
+  }
+
+  getRandom() { ... }
+
+  static entropy_pool;
+  static {
+    this.entropy_pool = [];
```

Comment:
Nit: If we're going to demonstrate a static field, we should just initialize with it.

```suggestion
  static entropy_pool = [];
  static {
    this.entropy_pool;
```

Reply:
Sure.